docs/reference/ClientIntents CRD/README.mdx

```diff
@@ -0,0 +1,120 @@
+---
+sidebar_position: 7
+title: 'ClientIntents CRD'
+---
+
+ORIORIORI
+
+```yaml
+apiVersion: k8s.otterize.com/v2alpha1
+kind: ClientIntents
+metadata:
+  name: intents-for-checkoutservice
+  namespace: otterize-ecom-demo
+spec:
+  workload:
+    name: checkoutservice # Name of the workload initiating calls
+    kind: Deployment # Type of workload (e.g., Deployment, StatefulSet, etc.)
+  targets:
+    - kubernetes:
+        name: emailservice # Target Kubernetes resource name
+        kind: Service # Kind of the Kubernetes resource
+    - service: # Equivalent to kubernetes target with kind: Service
+        name: orderservice # Target Service name
+        http:
+          - path: /orders # HTTP path to target
+            methods: [ GET ] # Allowed HTTP methods
+    - kafka:
+        name: kafka-prod # Kafka cluster name
+        topics:
+          - name: orders # Kafka topic
+            operations: [ read, write ] # Allowed operations on the topic
+    - sql:
+        name: mysql-db # SQL database resource name
+        privileges:
+          - databaseName: ecommerce # Target database name
+            table: orders # Table name
+            operations: [ SELECT, INSERT ] # Allowed SQL operations
+    - internet:
+        domains:
+          - api.example.com
+        ips:
+          - 192.168.1.1
+        ports:
+          - 443
+    - aws:
+        arn: arn:aws:s3:::my-bucket # AWS resource ARN
+        actions:
+          - s3:GetObject
+          - s3:PutObject
+    - gcp:
+        resource: projects/my-project-id/buckets/my-bucket
```

Comment:
Missing annotations for some lines in this file (picked one at random, but there are more, like Internet)

Reply:
Fixed